rasa/plugin.py

```diff
@@ -73,3 +73,8 @@ def create_tracker_store(  # type: ignore[empty-body]
     event_broker: Optional["EventBroker"],
 ) -> "TrackerStore":
     """Hook specification for wrapping with AuthRetryTrackerStore."""
+
+
+@hookspec  # type: ignore[misc]
+def get_license_hash() -> Optional[Text]:
```

Comment:
Could we please add a unit test for this (asserting that it was called in the relevant context)?